pod/urls.py

```diff
@@ -124,6 +125,9 @@
 
     # custom
     url(r'^custom/', include('pod.custom.urls')),
+
```

Comment:
I think it's better to put custom urls at the last of url_pattern. Can you move select2.urls just before custom urls ?